src/docs/truffle/getting-started/creating-a-project.md

```diff
@@ -29,6 +29,10 @@ You can create a bare project template, but for those just getting started, you
    <strong>Note</strong>: To create a bare Truffle project with no smart contracts included, use `truffle init`.
    </p>
 
+   <p class="alert alert-info">
+   <strong>Note</strong>: You can use an optional `--force` to initialize the project in the current directory regardless of its state (e.g. even if it contains other files or directories). This applies to both the `init` and `unbox` commands. Be careful, this will potentially overwrite files that exist in the directory.
```

Comment:
If you don't use the `--force` flag it will just prompt you for each file/folder name conflict and ask you whether you want to overwrite the pre-existing item.